packages/frontend/src/pages/settings-page/components/profile-settings/profile-settings.tsx

```diff
@@ -363,6 +367,15 @@ export const ProfileSettings = ({ user, onSubmit, isSubmitting }: Props) => {
             name="readme"
             defaultValue={user.readme}
             render={({ field: { onChange, value } }) => (
+              // TODO: MarkdownSplitEditor is not mobile-friendly
+              //
+              // On Mobile:
+              // 1. Enable Scroll Sync checkbox doesn't look good
+              // 2. Editor is not visisble at all unless the 'Scroll Sync' is enabled
+              //
+              // On Desktop:
+              // 1. Only shows 3 or 4 lines at a time no matter the window height
```

Comment:
Does your other PR address all of these?

Reply:
I verified this and it does 😄